packages/wonder-blocks-modal/components/modal-backdrop.test.js

```diff
@@ -88,10 +92,30 @@ describe("ModalBackdrop", () => {
 
         // Act
         await sleep(); // wait for styles to be applied
-        const lastButton = wrapper.find("[data-last-button]").getDOMNode();
+        const focusableElement = wrapper
+            .find("[data-first-button]")
```

Comment:
This `data` attribute is just for testing purposes right?

Reply:
yes, just for testing